TI-reports/2024/2024-Q3-BEST-WG.md

```diff
@@ -0,0 +1,130 @@
+Materials will be maximally accessible and easy to consume for all learners.
+#### Current Status
+- Many simultaneous activities
+- Recent release of LF Research study on Security Edutation for Developers
+- Academic Accredidation team working on kicking off program to "certify" collegiate programs that meet OpenSSF & CNCF best practices
+- Security for Developer Managers class progressing into two pieces of collateral: Manager class & terms-definitions
+#### Up Next
+- Security Architect class outline reviewed and content development will come next
+- "201 level" class will come after
+- 
+### Memory Safety SIG
+#### Purpose
+- The Memory Safety SIG is a group working within the OpenSSF's Best Practices Working Group formed to advance and deliver upon The OpenSSF's Mobilization Plan - Stream 4.
+#### Current Status
+-Have drafted a “Memory Safety Continuum” concept document
+- Have gathered guides/practices related to best memory safety practices in both memory safe by default and non memory safe by default languages
+#### Up Next 
+- Produce a Memory Safety workshop (modeled after W3C workshops). Theme is “Improving Memory Safety in an Imperfect World”
+- Finalize Memory Safety Continuum doc
+
+### Python Hardening Guide
+#### Purpose
+
+#### Current Status
+
+#### Up Next
+
+### Scorecard
+#### Purpose
+-To help open source maintainers improve their security best practices and to help open source consumers judge whether their dependencies are safe.
+- Scorecard is an automated tool that assesses a number of important heuristics ("checks") associated with software security and assigns each check a score of 0-10.
+#### Current Status
+
+#### Up Next
+
+
+### Security Baseline
+#### Purpose
+- The goal of this SIG is to evolve OpenSSF security baseline for Linux Foundation wide adoption.
+- For OpenSSF adoption of the security baseline, there needs to be a home for tracking the adoption, for maintainers to raise issues to refine the security baseline, merge the baseline back to TAC lifecycle, and for OpenSSF to develop the roadmap for the security baseline. It will provide a venue for early adopters to share their reusable code and findings with other maintainers. The pilot adoption builds the foundation for wider adoption of the security baseline in OpenSSF and in Linux Foundation.
+- This SIG creates a venue for other participating foundations to help evolve the OpenSSF security baseline into a security baseline that can be applied to a broad range of software-based projects. The group will define the right level of risks that the security baseline is applicable for, the effectiveness measurement of the security baseline, and the adoption path of the security baseline at the minimum.
+#### Current Status
+- on 16July the WG voted to adopt the OpenSSF Security Baseline as a SIG within our group.
```

Comment:
typo
```suggestion
- on 16 July the WG voted to adopt the OpenSSF Security Baseline as a SIG within our group.
```

Reply:
@SecurityCRob 5 Projects are adopting the security baseline now, 2 more than when you started the update.